pkg/scheduler/ugm/manager.go

```diff
@@ -221,6 +221,15 @@ func (m *Manager) GetUsersResources() []*UserTracker {
 	return userTrackers
 }
 
+func (m *Manager) GetUserResourcesByUserName(user string) *UserTracker {
```

Comment:
Can we rename the method name as GetUserResources?

Reply:
Hi @manirajv06 , we already have a function name this, so i name to GetUserResourcesByUserName
```
func (m *Manager) GetGroupResources(group string) *resources.Resource {
	m.lock.RLock()
	defer m.lock.RUnlock()
	if m.groupTrackers[group] != nil {
		return m.groupTrackers[group].queueTracker.resourceUsage
	}
	return nil
}
```